src/popup/components/AccountSelectOptionsItem.vue

```diff
@@ -89,7 +89,7 @@ export default defineComponent({
     @include mixins.flex(space-between, flex-start, row);
 
     position: relative;
-    padding: 8px;
+    padding: 5px 8px;
```

Comment:
```suggestion
    padding: 6px 8px;
```
Judging by element size in design. 
![image](https://github.com/aeternity/superhero-wallet/assets/9008074/ae36158d-bf8e-4574-8570-738027295cc0)
Current size 
![image](https://github.com/aeternity/superhero-wallet/assets/9008074/1f6bcb92-c6e0-4c06-b817-7f719f718f1c)